### PR TITLE
Adding ODM annotation @ODM.oidReference.entityName

### PR DESCRIPTION
**Pull request by ArkadiyPonomarov**
Added

- ODM annotation @ODM.oidReference.entityName (same name in CDS)

### CODE REVIEW

**Comment by ArkadiyPonomarov** [comment]
@ralfhandl Could you please review?

**Review by ralfhandl — CHANGES_REQUESTED** [review]
Looks good, just add "Experimental" on both term and property until it has been implemented by SAP NetWeaver AS ABAP

**Comment by ArkadiyPonomarov** [comment]
Changes are done, thank you.

**Review by ralfhandl — APPROVED** [review]

File: vocabularies/ODM.xml [hunk 1]
```diff
@@ -60,6 +60,21 @@ An entity may have other IDs, i.e. a local ID and other alternative IDs. </Strin
         </Annotation>
       </Term>
 
+      <Term Name="oidReference" Type="ODM.oidReferenceType" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />      
+        <Annotation Term="Core.Description" String="Specification of an referenced entity" />
```
Comment (HeikoTheissen) [inline vocabularies/ODM.xml:65]:
```suggestion
        <Annotation Term="Core.Description" String="Specification of a referenced entity" />
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

**Review by HeikoTheissen — DISMISSED** [review]

**Review by ralfhandl — APPROVED** [review]

**Comment by ArkadiyPonomarov** [comment]
Hi colleagues, Is there any stopper? If not, could you please review/approve?

**Review by martin4all — APPROVED** [review]

**Pull request merged**